### DISCUSSION TITLE
How to evaluate the solution at other points (locations) using GridFunction object

### DISCUSSION THREAD

**Discussion by JianboLong** [mfem/mfem#3940] [General]
@tzanio  My question is, after solving the linear system, the GridFunction object has the solution of all degrees of freedom. How can we get solutions at other locations defined by the user ? 

Thanks !

**Comment by tzanio** [comment]
You can use MFEM's findpts functionality, see miniapps/gslib/findpts.cpp.

cc: @kmittal2

**Reply by JianboLong** [reply]
Thanks @tzanio ! I'm going through the details of this miniapp, which looks like exactly what I want to do for post-processing of FEM. I wonder is there other similar examples so that I can understand this miniapp better ?

**Reply by kmittal2** [reply]
Hi @JianboLong, `findpts` and `pfindpts` are the two basic miniapps demonstrating use of `FindPointsGSLIB` for interpolating a `GridFunction`. Another useful resource is from the HowTo section of the MFEM website that gives an overview of different key functions in the `FindPointsGSLIB` class: https://mfem.org/howto/findpts/.
